decompiler/config/jak2/all-types.gc

```diff
@@ -8111,15 +8111,16 @@
 ;; +++main-h:scene-controls
 (defenum scene-controls
   :type int64
-  (all-off 0)
-  (channel 1)
-  (anim-name 2)
-  (dma-size 4)
-  (bounds-spheres 8)
-  (actors 16)
-  (actor-marks 32)
-  (special-fma-spheres 64)
-  (all-on 95))
+  :bitfield #t
+  (all-off)
+  (channel)
+  (anim-name)
+  (dma-size)
+  (bounds-spheres)
+  (actors)
+  (actor-marks)
+  (special-fma-spheres)
+  (all-on))
```

Comment:
Is this change right? `all-off` will now be 1 instead of 0, `channel` will be 2... and `all-on` is definitely not 95 any more.

Reply:
Yeah I'm not so sure about this either...but the naming came from the debug menu code https://github.com/open-goal/jak-project/blob/master/goal_src/jak2/engine/debug/default-menu.gc#L5579-L5580 Perhaps I'm interpreting it wrong / it's not related?